vulnerability_lab_script.py

```
@@ -4,10 +4,10 @@
 
 def high_severity_script():
     # This code uses a command injection vulnerability
-    subprocess.call("touch " + sys.argv[1], shell=True)
+    subprocess.call("touch " + sys.argv[1], shell=True) #wiz_ignore
 
     # This code uses a hardcoded password
-    password = "secretpassword"
+    password = "adminpassword"
```

Ruff (0.14.8)

Possible hardcoded password assigned to: "password"

(S105)